### ISSUE TITLE
Error message on startup when video not enabled (Trac #93)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#97]
ERROR: microdrop.gui.dmf_device_controller plugin crashed processing on_app_options_changed signal.
Reason: 'NoneType' object has no attribute 'get_allocation'
C:\buildbot\microdrop\WinXP_Py2_7\build\build\pyi.win32\microdrop\outPYZ1.pyz/gui.main_window_controller:203: GtkWarning: gdk_property_change: assertion `window
 != NULL' failed

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/93
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "ERROR: microdrop.gui.dmf_device_controller plugin crashed processing on_app_options_changed signal.\nReason: 'NoneType' object has no attribute 'get_allocation'\nC:\\buildbot\\microdrop\\WinXP_Py2_7\\build\\build\\pyi.win32\\microdrop\\outPYZ1.pyz/gui.main_window_controller:203: GtkWarning: gdk_property_change: assertion `window\n != NULL' failed", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Error message on startup when video not enabled", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-04-03T19:57:51", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120403T21:12:17: 
* ryan@fobel.net commented:

> commit 758fd101ae26adcf324d1fcf4b8b47ecc0e7a8fe
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Tue Apr 3 17:08:20 2012 -0400
> 
>     Fix #97 (startup error)


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
